src/AutoMapper/Internal/TypeHelper.cs

```diff
@@ -1,169 +0,0 @@
-        }
-
-        public static bool HasAnInaccessibleSetter(PropertyInfo property)
-        {
-            var setMethod = property.GetSetMethod(true);
-            return setMethod == null || setMethod.IsPrivate || setMethod.IsFamily;
-        }
-
-        public static bool IsPublic(MemberInfo memberInfo) => (memberInfo as FieldInfo)?.IsPublic ?? (memberInfo as PropertyInfo).IsPublic();
-
-        public static bool IsNotPublic(ConstructorInfo constructorInfo) => constructorInfo.IsPrivate
-                                                                                || constructorInfo.IsFamilyAndAssembly
-                                                                                || constructorInfo.IsFamilyOrAssembly
-                                                                                || constructorInfo.IsFamily;
-
-        public static Assembly Assembly(Type type) => type.GetTypeInfo().Assembly;
-
-        public static Type BaseType(Type type) => type.GetTypeInfo().BaseType;
-
-        public static bool IsAssignableFrom(Type type, Type other) => type.GetTypeInfo().IsAssignableFrom(other.GetTypeInfo());
-
-        public static bool IsAbstract(Type type) => type.GetTypeInfo().IsAbstract;
-
-        public static bool IsClass(Type type) => type.GetTypeInfo().IsClass;
-
-        public static bool IsEnum(Type type) => type.GetTypeInfo().IsEnum;
-
-        public static bool IsGenericType(Type type) => type.GetTypeInfo().IsGenericType;
-
-        public static bool IsGenericTypeDefinition(Type type) => type.GetTypeInfo().IsGenericTypeDefinition;
-
-        public static bool IsInterface(Type type) => type.GetTypeInfo().IsInterface;
-
-        public static bool IsPrimitive(Type type) => type.GetTypeInfo().IsPrimitive;
-
-        public static bool IsSealed(Type type) => type.GetTypeInfo().IsSealed;
-
-        public static bool IsValueType(Type type) => type.GetTypeInfo().IsValueType;
-
-        public static bool IsInstanceOfType(Type type, object o) => o != null && type.GetTypeInfo().IsAssignableFrom(o.GetType().GetTypeInfo());
-
-        public static ConstructorInfo[] GetConstructors(Type type) => type.GetTypeInfo().DeclaredConstructors.ToArray();
-
-        public static PropertyInfo[] GetProperties(Type type) => type.GetRuntimeProperties().ToArray();
-
-#if NET40
-        public static MethodInfo GetGetMethod(PropertyInfo propertyInfo, bool ignored) => propertyInfo.GetGetMethod();
-
-        public static MethodInfo GetSetMethod(PropertyInfo propertyInfo, bool ignored) => propertyInfo.GetSetMethod();
-#else
-        public static MethodInfo GetGetMethod(PropertyInfo propertyInfo, bool ignored) => propertyInfo.GetMethod;
-
-        public static MethodInfo GetSetMethod(PropertyInfo propertyInfo, bool ignored) => propertyInfo.SetMethod;
-#endif
-
-        public static FieldInfo GetField(Type type, string name) => type.GetRuntimeField(name);
-    }
-}
```

Comment:
I don't see what this has to do with my PR - I guess you should have created another branch/PR for this, but ok ;-)